addons/point_of_sale/static/src/@types/services.d.ts

```diff
@@ -1,16 +1,25 @@
 declare module "services" {
+    import { alertService } from "@point_of_sale/app/services/alert_service";
     import { barcodeReaderService } from "@point_of_sale/app/services/barcode_reader_service";
+    import { contextualUtilsService } from "@point_of_sale/app/services/contextual_utils_service";
     import { debugService } from "@point_of_sale/app/services/debug_service";
     import { hardwareProxyService } from "@point_of_sale/app/services/hardware_proxy_service";
     import { numberBufferService } from "@point_of_sale/app/services/number_buffer_service";
-    import { notificationService } from "@point_of_sale/app/notification/notification_service";
+    import { PosDataService } from "@point_of_sale/app/services/data_service";
+    import { printerService } from "@point_of_sale/app/services/printer_service";
```

Comment:
```suggestion
    import { posPrinterService } from "@point_of_sale/app/services/pos_printer_service";
```

Isn't it posPrinterService that we should use here ?